documentation/largescaledeployment.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Recommended configuration for large scale cluster deployment
+---
+Kubernetes vSphere Cloud Provider is tested with 500 nodes.
+
+Below is the cluster configuration recommended for 500 node cluster.
+
+**Node CPU and Memory Capacity Allocation**
+
+To handle workload for 500 node Kubernetes cluster master node is configured to use 8 cpus and 8 GB memory.
+
+```
+Capacity:
+ cpu:     8
+ memory:  8174428Ki
+```
+**Note:** Here 8 CPU cores means 8000 milli-cores.
+
+Worker nodes are allocated 2 GB and 2 CPUs.
+
+**Note:** CPU/Memory allocation for nodes can be adjusted based on the workload requirement. Above is the minimum recommended allocation.
```

Comment:
Above is the minimum required allocation.

Reply:
Fixed.